Formula/ice.rb

```diff
@@ -58,6 +58,7 @@ def install
     args = %W[
       prefix=#{prefix}
       USR_DIR_INSTALL=yes
+      SLICE_DIR_SYMLINK=yes
```

Comment:
@externl no problem. I installed locally to make sure I understood what you meant, and this looks like a reasonable solution for now.